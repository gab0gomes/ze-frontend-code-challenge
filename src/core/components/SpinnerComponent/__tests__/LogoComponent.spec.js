import { mount, createLocalVue } from '@vue/test-utils'

import Spinner from '../index'

const localVue = createLocalVue()

describe('SpinnerComponent', () => {
  it('render correctly', async () => {
    const wrapper = mount(Spinner, {
      localVue
    })

    expect(wrapper.element).toMatchSnapshot()

    wrapper.destroy()
  })
})
