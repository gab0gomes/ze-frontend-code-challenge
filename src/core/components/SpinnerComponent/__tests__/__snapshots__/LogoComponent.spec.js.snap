// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SpinnerComponent render correctly 1`] = `
<section
  class="spinner-container"
>
  <div
    class="spinner"
  />
</section>
`;
